### DISCUSSION TITLE
Exclude some fields like createdAt for CreateInputSchema

### DISCUSSION THREAD

**Discussion by Extarys** [chrishoermann/zod-prisma-types#146] [Q&A] [answered]
Is there a way to exclude some fields for *CreateInputSchema?
I noticed that `createdAt` and `updatedAt` are there as optional in the schema, but those type of fields shouldn't be in the Input Schema since it's the system that manages them.

**Comment by chrishoermann** [comment, marked as answer]
Hey @Extarys, did you try to use 

```prisma
createdAt Date /// zod.custom.omit([input])
```

as stated in [the docs](https://github.com/chrishoermann/zod-prisma-types#omit-fields)?

**Reply by Extarys** [reply]
Yes sorry, I found that out yesterday and forgot to update here. I added it throughout my schema :smile: Thanks for stepping in though.
